docs/dev/antora.yml

```diff
@@ -9,7 +9,7 @@ content:
 
 ui:
   bundle:
-    url: https://lightbend.github.io/antora-supplemental-ui-lightbend-theme/default_ui/ui-bundle-1023155878.zip
```

Comment:
not sure, but compare with Javascript https://github.com/lightbend/akkaserverless-javascript-sdk/pull/195

Reply:
Tried building the java docs, and that preview doesn't show the tabs either (and also didn't get an updated theme version). Maybe there was no new zip with the tabs created, or maybe it needs more infra to actually show tabs. Not super important just a minor annoyance.

Reply:
@pvlugter or @ennru Do you know what needs to be done to enable tabs in the javasdk docs? I assume we already have that working in javascript?

Reply:
To be clear: I'm expecting that it works when it is pulled into the main docs site but not when building and previewing the respective sdks (didn't try and verify though).

Reply:
I can confirm that it's looking good when building from akkaserverless-docs using 
```
antora --stacktrace local.yml && cp get.sh build/site/
```

Reply:
Yes, would be good to have the actual theme here in the SDK docs. For JS docs, I just viewed the main docs locally. Haven't looked at it properly yet, but akkaserverless-docs seems to have a bunch of things locally where it would be better to have everything in a published bundle.